packages/fm-player/src/components/SoundBoard.tsx

```diff
@@ -5,7 +5,9 @@ import SoundButton from './SoundButton';
 import Search from './Search/Search';
 
 function soundMatchesPhrase(sound: XSound, phrase: string) {
-  return sound.name.includes(phrase) || (sound.tags || []).some((tag) => tag.includes(phrase));
+  const { name, tags } = sound;
+  const tagsAndNameJoinedString = [name, ...tags ?? []].join('').toLowerCase();
```

Comment:
I would use `join(' ')` with a space inside, otherwise we may get a match for a phrase that consists of last letters of name concatenated with first letters of a tag.

Reply:
Nice catch. I used "" to shorten the search string but did not thought about the case you mentioned.